js/base/main.js

```diff
@@ -283,8 +283,10 @@ export class RipeCommonsMainPlugin extends RipeCommonsPlugin {
             // runs the remote business logic to obtain the multiple
             // target groups available for initials as well as the
             // available characters for personalization
-            const [groups, supportedCharacters] = await Promise.all([
+            const [groups, minCharacters, maxCharacters, supportedCharacters] = await Promise.all([

```

Comment:
Let's use the same naming that exists already (minimum_initials & maximum_initials)